Src/FluentAssertions/Primitives/StringAssertions.cs

```diff
@@ -678,10 +678,10 @@ public AndConstraint<TAssertions> Contain(string expected, OccurrenceConstraint
             int actual = Subject.CountSubstring(expected, StringComparison.Ordinal);
 
             Execute.Assertion
-                .ForCondition(occurrenceConstraint.Assert(actual))
+                .ForOccurrences(occurrenceConstraint, actual)
```

Comment:
I think `ForConstraint` will work better, but I'm not 100% sure. I even ask [Twitterverse](https://twitter.com/ddoomen/status/1267006156825931776) for some guidance.

Reply:
I guess that guys in Twitter aren't actually getting the context for these concepts. IMHO `ForConstraint` will work just fine, it's correct in technical sense.